src/share/element_mod.F90

```diff
@@ -835,9 +835,7 @@ subroutine setup_element_pointers(elem)
     allocate( elem_derived_dp               (np,np,nlev,nelemd)                    )
     allocate( elem_derived_divdp            (np,np,nlev,nelemd)                    )
     allocate( elem_derived_divdp_proj       (np,np,nlev,nelemd)                    )
-#ifndef USE_KOKKOS_KERNELS
     allocate( elem_derived_FQ               (np,np,nlev,qsize_d,nelemd) )
```

Comment:
could this have been affecting performance of hommexx in a better way? would not having this allocated make hommexx faster?